packages/edit-post/src/components/header/index.js

```diff
@@ -107,10 +108,9 @@ function Header( { setEntitiesSavedStatesCallback, initialPost } ) {
 			>
 				<DocumentTools disableBlockTools={ isTextEditor } />
 				{ hasTopToolbar && (
-					<ContextualToolbar
+					<BlockContextualToolbar
 						isCollapsed={ isBlockToolsCollapsed }
-						blockSelectionStart={ blockSelectionStart }
-						toggleCollapse={ handleToggleCollapse }
+						onToggle={ handleToggleCollapse }
```

Comment:
We can pass `setIsBlockToolsCollapsed` here (like in `edit-site`) and remove `handleToggleCollapse`.

Reply:
Good catch.